web/themes/custom/fmcg_material/sass/styles.scss

```diff
@@ -2273,3 +2098,100 @@ form#views-exposed-form-job-board-page {
     border-radius: 2px;
   }
 }
+
+@media (min-width: 1256px) {
+  .navbar-hamburger-menu {
+    display: none !important;
+  }
+}
+
+.path-frontpage div#main-content, .path-user div#main-content, .path-user-login div#main-content {
+  background: none;
+}
+
+.password-strength__title, .password-confirm {
+  font-family: Roboto;
+  font-size: 13px;
+  color: #DBE4EB;
+}
+
+.password-strength__meter {
+  height: 0.75em;
+  margin-top: 0.5em;
+  background-color: #211F21;
+}
+
+.password-strength__indicator {
+  width: 0;
+  height: 100%;
+  background-color: #FDD22B;
+}
+
+.job-post-organization-form {
```

Comment:
Is this really only for this form? It seems like really we want all forms to be the saem *apart from* the login form - and maybe register.